content/en/docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -237,24 +255,55 @@ used to verify the serving certificate of the API server.
 Finally, the default namespace to be used for namespaced API operations is placed in a file
 at `/var/run/secrets/kubernetes.io/serviceaccount/namespace` in each container.
 
-From within a Pod, the recommended ways to connect to the Kubernetes API are:
+#### Using kubectl proxy
 
-  - For a Go client, use the official [Go client library](https://github.com/kubernetes/client-go/).
-    The `rest.InClusterConfig()` function handles API host discovery and authentication automatically.
-    See [an example here](https://git.k8s.io/client-go/examples/in-cluster-client-configuration/main.go).
+If you would like to query the API without an official client library, you can run `kubectl proxy`
+as the [command](/docs/tasks/inject-data-application/define-command-argument-container/)
+of a new sidecar container in the Pod. This way, `kubectl proxy` will authenticate
+to the API and expose it on the `localhost` interface of the Pod, so that other containers
+in the Pod can use it directly.
 
-  - For a Python client, use the official [Python client library](https://github.com/kubernetes-client/python/).
-    The `config.load_incluster_config()` function handles API host discovery and authentication automatically.
-    See [an example here](https://github.com/kubernetes-client/python/blob/master/examples/in_cluster_config.py).
-    
-  - If you would like to query the API without an official client library, you can run `kubectl proxy`
-    as the [command](/docs/tasks/inject-data-application/define-command-argument-container/)
-    of a new sidecar container in the Pod. This way, `kubectl proxy` will authenticate
-    to the API and expose it on the `localhost` interface of the Pod, so that other containers
-    in the Pod can use it directly.
+#### Without using a proxy
 
-In each case, the service account credentials of the Pod are used to communicate
-securely with the API server.
+It is possible to avoid using the kubectl proxy by passing the authentication token
+directly to the API server.  The internal certificate secures the connection.
+
+```shell
+# Point to the internal API server hostname
+APISERVER=https://kubernetes.default.svc
+
+# Serviceaccount location
+SERVICEACCOUNT=/var/run/secrets/kubernetes.io/serviceaccount
+
+# Reads the default namespace
+NAMESPACE=$(cat ${SERVICEACCOUNT}/namespace)
+
+# Gets the bearer token value
+TOKEN=$(cat ${SERVICEACCOUNT}/token)
+
+# Reference the internal CA
```

Comment:
```suggestion
# Reference the internal certificate authority (CA)
```

Reply:
I wouldn't be too pedantic about these, my starting point was the exsiting example further up the page...